src/main/java/org/broadinstitute/consent/http/models/User.java

```diff
@@ -188,6 +189,36 @@ public void setRoles(List<UserRole> roles) {
     this.roles = roles;
   }
 
+  public void setAdminRole() {
+    this.roles = Collections.singletonList(UserRoles.AdminRole());
```

Comment:
In case you like it better, Java 9+ includes `List.of(..)` method, which has been my personal favorite for defining immutable lists, singleton or otherwise.

Reply:
I'm generally in favor of `List.of(..)` as well. The one caveat I've had with refactoring to that has to do with tests that attempt list mutations.